content/en/docs/setup/install/helm/index.md

```diff
@@ -15,10 +15,14 @@ icon: helm
 window.onload = function(){
```

Comment:
I'd like this to be removed please.

The intent with the tabs is that they are sticky. The user sets them and they will remain set that way whenever the user visits the page. This overriding behavior is counter to that.

Reply:
Done: removed all changes in this file.